src/wallet/utils/isBase.ts

```diff
@@ -0,0 +1,15 @@
+import { isBaseOptions, isBaseResponse } from '../types';
+import { baseSepolia, base } from 'viem/chains';
+
+/**
+ * isBase
+ *  - Checks if the paymaster operations chain id is valid
+ *  - Only allows the base and baseSepolia chain ids
+ */
+export function isBase({ chainId }: isBaseOptions): isBaseResponse {
```

Comment:
Write a doc page as well in the `site` folder please.

And add a changeset patch as well.

Reply:
Added docs page and changeset!